faces/wildfly-mods/se21-excludes-profile.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+        <profile xmlns="http://maven.apache.org/POM/4.0.0">
+			<!-- Exclude accepted TCK challenges that are only relevant to SE 21 or later. -->
+            <id>se21-tck-challenges</id>
+            <activation>
+                <jdk>[21,)</jdk>
+				<property><name>se21.newtck.exclusions</name></property>
+            </activation>
+
+            <build>
+            	<plugins>
+            	    <plugin>
+            	        <groupId>org.apache.maven.plugins</groupId>
+            	        <artifactId>maven-failsafe-plugin</artifactId>
+            	        <configuration>
+            	            <excludesFile>${se21.newtck.exclusions}</excludesFile>
```

Comment:
Note I went with an excludesFile element here instead of an excludes element because it's not clear from the failsafe documentation whether method filtering is supported for 'excludes'. It's only documented in the excludesFile row:

`Since 3.0.0-M6, method filtering support is provided in the exclusions file as well, example:`

If it turns out it works with 'excludes' as well (which seems reasonable) in a future improvement we can perhaps switch to that and remove a couple files.

Reply:
A file seems fine as it can be updated in a more clear manor IMO.